invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/RenameCommunitySlugButton.js

```diff
@@ -65,6 +65,8 @@ export class RenameCommunitySlugButton extends Component {
         this.setState({ error: invalidIdError });
       }
     }
+
+    this.setState({ loading: false });
```

Comment:
sorry, not a part of the scope but I just noticed possible memory leak in line 54 and 55, could you swap them ? Normally we should first handle close event and then change the window location